Dfc.CourseDirectory/Resources/ArmTemplates/parameters.json

```diff
@@ -14,6 +14,12 @@
     "postcodeApiKey": {
       "value": "__postcodeApiKey__"
     },
+    "getAddressApiUrl": {
```

Comment:
Why is the URL in the ARM template?

Reply:
The Loqate one was, but it wasn't used by the implementation. Typically it's good practice to have these values configurable so if they need to change they can be updated without the need for a new release, equally it gives us the option to use a different endpoint for different environments should the need arise.

Reply:
So it can live in `appsettings.json`

Reply:
Potentially, yes.